### ISSUE TITLE
24.04

### ISSUE THREAD

**Issue by KhensU** [pipewire-debian/pipewire-debian#110]
Any chance of getting pipewie 1.2.3 for 24.04 ?  Just a friendly request.

**Comment by nemoforum** [comment]
Not pushing, just curious about Pipewire 1.2x and libcamera 0.3.x. I'm interested in this because Linux 6.10 includes an Intel IPU6 Driver for modern web cameras and libcamera 0.3.x is needed.
Thank you for your work!

**Comment by ashleysommer** [comment]
I'm in the same boat, I'm on Ubuntu 22.04 with the latest backported HWE kernel, it supports IPU6 cameras, but I need libcamera 0.3.x.

**Comment by user3354** [comment]
If you are still using 22.04, have a look at Rob Savoury's PPA.

**Comment by KhensU** [comment]
It looks like Rob Savoury's repo just updated with 1.2.3 for 24.04.

**Comment by user3354** [comment]
Yes, if you are using Ubuntu LTS or Linux Mint you are now covered.
